### PR TITLE
pages.zh: update outdated pages

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.zh/common/ar.md [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
 
 - 列出库文件中的成员：
 
-`ar -t {{a 文件}}`
+`ar t {{a 文件}}`
 
 - 替换或添加文件到库文件：
 
-`ar -r {{要被添加内容的 a 文件}} {{o 文件 1}} {{o 文件 2}} {{o 文件 3}}`
+`ar r {{要被添加内容的 a 文件}} {{o 文件 1 o 文件 2 o 文件 3 ...}}`
```
Comment (blueskyson) [inline pages.zh/common/ar.md:16]: Correspond to file1 file2 ...
```suggestion
`ar r {{要被添加内容的 a 文件}} {{o 文件1 o 文件2 o 文件3 ...}}`
```

File: pages.zh/common/ar.md [hunk 2]
```diff
@@ -5,20 +5,20 @@
...
 
 - 插入对象文件索引（相当于使用`ranlib`）：
 
-`ar -s {{a 文件}}`
+`ar s {{a 文件}}`
 
 - 使用文件和附带的目标文件索引创建存档：
 
-`ar -rs {{a 文件}} {{o 文件 1}} {{o 文件 2}} {{o 文件 3}}`
+`ar rs {{a 文件}} {{o 文件 1 o 文件 2 ...}}`
```
Comment (blueskyson) [inline pages.zh/common/ar.md:24]:
```suggestion
`ar rs {{a 文件}} {{o 文件1 o 文件2 ...}}`
```

File: pages.zh/common/case.md [hunk 3]
```diff
@@ -5,8 +5,8 @@
 
 - 通过字符串字面量判断执行分支：
 
-`case {{入参变量}} in {{字符字面量1}} {{执行语句块1}} ;; {{字符字面量2}}) {{执行语句块2}} ;; *) {{默认执行语句块}} ;; esac`
+`case {{入参变量}} in {{字符字面量1}}) {{wc -w 执行语句块1}}; ;; {{線}}) {{wc -l 执行语句块}}; ;; esac`
```
Comment (blueskyson) [inline pages.zh/common/case.md:8]: It's just a translation fix. I don't even understand the example in English.
```suggestion
`case {{入参变量}} in {{字符字面量}}) {{wc -w 执行语句块}}; ;; {{字符串}}) {{wc -l 执行语句块}}; ;; esac`
```

File: pages.zh/common/case.md [hunk 4]
```diff
@@ -5,8 +5,8 @@
...
 
 - 搭配通配符进行匹配，判断执行分支：
 
-`case {{入参变量}} in {{通配符或者字符字面量}}) {{执行语句块1}} ; ;; {{通配符或者字符字面量}}) {{执行语句块1}}; ;; *) {{echo "what?"}}; ;; esac`
+`case {{入参变量}} in {{通配符或者字符字面量}}) {{执行语句块1}}; ;; {{通配符或者字符字面量}}) {{执行语句块1}}; ;; *) {{echo "what?"}}; ;; esac`
```
Comment (blueskyson) [inline pages.zh/common/case.md:12]:
```suggestion
`case {{入参变量}} in {{[wW]|字符字面量}}) {{wc -w 执行语句块}}; ;; {{[lL]|字符串}}) {{执行语句块}}; ;; *) {{echo "what?"}}; ;; esac`
```

File: pages.zh/common/find.md [hunk 5]
```diff
@@ -21,15 +21,15 @@
 
 - 查找符合指定大小范围的文件：
```
Comment (blueskyson) [inline pages.zh/common/find.md:22]:
```suggestion
- 查找符合指定大小范围的文件，将递归深度限制为 "1"：
```

File: pages.zh/common/find.md [hunk 6]
```diff
@@ -21,15 +21,15 @@
...
 
-`find {{指定目录}} -size {{+500k}} -size {{-10M}}`
+`find {{指定目录}} -maxdepth 1 -size {{+500k}} -size {{-10M}}`
 
 - 对每个文件运行命令（在命令中使用 `{}` 代表当前文件）：
 
 `find {{指定目录}} -name '{{*.ext}}' -exec {{wc -l {} }}\;`
 
 - 查找最近 7 天修改的文件并删除：
```
Comment (blueskyson) [inline pages.zh/common/find.md:30]:
```suggestion
- 查找最近 7 天修改的文件：
```

File: pages.zh/common/mkdir.md [hunk 7]
```diff
@@ -3,14 +3,10 @@
 > 创建目录。
 > 更多信息：<https://www.gnu.org/software/coreutils/mkdir>.
 
-- 在当前目录或给定路径下创建一个目录：
-
-`mkdir {{目录}}`
-
 - 在当前目录下创建多个目录：
 
 `mkdir {{目录_1 目录_2 ...}}`
```
Comment (blueskyson) [inline pages.zh/common/mkdir.md:8]:
```suggestion
`mkdir {{路径/到/目录1 路径/到/目录2 ...}}`
```

File: pages.zh/common/mkdir.md [hunk 8]
```diff
@@ -3,14 +3,10 @@
...
 
 - 递归地创建目录（对创建嵌套目录很有用）：
 
-`mkdir -p {{路径/到/目录}}`
+`mkdir -p {{目录_1 目录_2 ...}}`
```
Comment (blueskyson) [inline pages.zh/common/mkdir.md:12]:
```suggestion
`mkdir -p {{路径/到/目录1 路径/到/目录2 ...}}`
```

File: pages.zh/common/mvn.md [hunk 9]
```diff
@@ -30,7 +30,7 @@
 
 - 执行清理操作并使用给定的构建配置打包代码，比如 `profileId` 如果有 dev、test、pro，可以指定其中一个 `profileId` 用来选择具体执行环境：
```
Comment (blueskyson) [inline pages.zh/common/mvn.md:31]:
```suggestion
- 清理然后使用给定的构建配置文件打包代码：
```

File: pages.zh/common/mvn.md [hunk 10]
```diff
@@ -30,7 +30,7 @@
...
 
-`mvn clean -P{{profileId}} package`
+`mvn clean -P {{profileId}} package`
```
Comment (blueskyson) [inline pages.zh/common/mvn.md:33]:
```suggestion
`mvn clean -P {{构建配置}} package`
```

File: pages.zh/common/tar.md [hunk 11]
```diff
@@ -6,11 +6,11 @@
 
 - 创建存档并将其写入文件：
 
-`tar cf {{target.tar}} {{file1}} {{file2}} {{file3}}`
+`tar cf {{target.tar}} {{file1 file2 ...}}`
```
Comment (blueskyson) [inline pages.zh/common/tar.md:9]: These terms can be translated later in other PR:
target -> `目标`, source -> `来源`.
```suggestion
`tar cf {{目标.tar}} {{文件1 文件2 ...}}`
```

File: pages.zh/windows/comp.md [hunk 12]
```diff
@@ -14,7 +14,7 @@
 
 - 比较两个文件集：
 
-`comp {{目录 1/*}} {{目录 2/*}}`
+`comp {{目录 1}}\* {{目录 }}\*`
```
Comment (blueskyson) [inline pages.zh/windows/comp.md:17]: The translation of this page can be modified later as well.
```suggestion
`comp {{路径\到\目录1}}\* {{路径\到\目录2}}\*`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for the comprehensive refresh. I left some suggestions.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**